tests/fixtures/TagLib.Tests.FileFormats/StandardTests.cs

```diff
@@ -84,48 +100,55 @@ public static void SetTags (Tag tag, TestTagLevel level = TestTagLevel.Normal)
 			tag.Composers = new string [] {"TEST composer 1", "TEST composer 2"};
 			tag.Conductor = "TEST conductor";
 			tag.Copyright = "TEST copyright";
+			tag.DateTagged = new DateTime(2017, 09, 12, 22, 47, 42);
 			tag.Disc = 100;
 			tag.DiscCount = 101;
 			tag.Genres = new string [] {"TEST genre 1", "TEST genre 2"};
 			tag.Grouping = "TEST grouping";
 			tag.Lyrics = "TEST lyrics 1\r\nTEST lyrics 2";
-			tag.Performers = new string [] {"TEST performer 1", "TEST performer 2"};
+			tag.Performers = new string[] { "TEST performer 1", "TEST performer 2" };
+			tag.PerformersRole = new string[] { "TEST role 1a; TEST role 1b", "TEST role 2" };
 			tag.Title = "TEST title";
+			tag.Subtitle = "TEST subtitle";
+			tag.Description = "TEST description";
 			tag.Track = 98;
 			tag.TrackCount = 99;
 			tag.Year = 1999;
-
 		}
 
 		public static void CheckTags (Tag tag, TestTagLevel level = TestTagLevel.Normal)
 		{
-			if (level >= TestTagLevel.Medium)
-			{
-				Assert.AreEqual("title sort, TEST", tag.TitleSort);
-				Assert.AreEqual("album sort, TEST", tag.AlbumSort);
-				Assert.AreEqual("performer sort 1, TEST; performer sort 2, TEST", tag.JoinedPerformersSort);
-				Assert.AreEqual("composer sort 1, TEST; composer sort 2, TEST", string.Join("; ", tag.ComposersSort));
-				Assert.AreEqual("album artist sort 1, TEST; album artist sort 2, TEST", string.Join("; ", tag.AlbumArtistsSort));
-			}
 
 			Assert.AreEqual ("TEST album", tag.Album);
 			Assert.AreEqual ("TEST artist 1; TEST artist 2", tag.JoinedAlbumArtists);
-			Assert.AreEqual (120, tag.BeatsPerMinute);
 			Assert.AreEqual ("TEST comment", tag.Comment);
 			Assert.AreEqual ("TEST composer 1; TEST composer 2", tag.JoinedComposers);
 			Assert.AreEqual ("TEST conductor", tag.Conductor);
-			Assert.AreEqual ("TEST copyright", tag.Copyright);
+			Assert.AreEqual("TEST copyright", tag.Copyright);
 			Assert.AreEqual (100, tag.Disc);
 			Assert.AreEqual (101, tag.DiscCount);
 			Assert.AreEqual ("TEST genre 1; TEST genre 2", tag.JoinedGenres);
 			Assert.AreEqual ("TEST grouping", tag.Grouping);
 			Assert.AreEqual ("TEST lyrics 1\r\nTEST lyrics 2", tag.Lyrics);
 			Assert.AreEqual ("TEST performer 1; TEST performer 2", tag.JoinedPerformers);
-			Assert.AreEqual ("TEST title", tag.Title);
+			Assert.AreEqual("TEST title", tag.Title);
```

Comment:
There should be a space between the method name and the opening `(`.

Reply:
Ok, I'll fix that.

Reply:
Fixed !